resources/lib/playbackmanager.py

```diff
@@ -26,8 +26,12 @@ def log(self, msg, level=2):
         ulog(msg, name=self.__class__.__name__, level=level)
 
     def launch_up_next(self):
-        playlist_item = True
+        playlist_item = True if get_setting('enablePlaylist') == "True" else False
```

Comment:
This line failed CI because pylint doesn't like the inline if-test, however just using `bool()` is not an option since the function returns a string (`"True"` or `"False"`) which always evaluates to `True`.
There used to be a similar function that would check for a `.bool` suffix added to the requested setting's name and return a Boolean value instead of a String